### DISCUSSION TITLE
Power usage?

### DISCUSSION THREAD

**Discussion by diogenes500** [sinara-hw/Kasli-SOC#79] [Q&A]
Is there a known, or even estimated, value for Kasli-SoC's and Kasli 's power usage?

**Comment by maciejprzybysz** [comment]
Power budget for Kasli could be found on schematic notes. This is maximum power required, but in fact it depends on your application. Please remember that also EEMs connected to Kasli are powered through EEM connectors, so you should also know power budget for whole system to estimate power source.

![2024-03-06_10h58_18](https://github.com/sinara-hw/Kasli-SOC/assets/48446953/6649f810-a566-431d-ab93-de56926e5b7c)

**Reply by maciejprzybysz** [reply]
Kasli SoC:
![2024-03-06_11h04_25](https://github.com/sinara-hw/Kasli-SOC/assets/48446953/13a9b4a3-7fb7-4f79-9693-ccbcf3265913)
